scripts/netdep_graph_json.py

```diff
@@ -46,6 +46,7 @@ def main():
   with open('clovis-trace.log') as f:
```

Comment:
Yeah okay... I'm not sure. Feels like we should keep it around in a separate branch on origin, but I don't want to merge it in unless we know we need it.